checker/src/org/checkerframework/checker/index/IndexChecker.java

```diff
@@ -13,7 +13,9 @@
  * <p>Constant Value Checker, SameLen Checker, SearchIndex Checker, Lower Bound Checker, Upper Bound
  * Checker
  *
- * <p>The Constant Value Checker has no dependencies.
+ * <p>The Constant Value Checker has no dependencies, but it does trust Positive annotations from
```

Comment:
Looking at ValueChecker.java, it doesn't mention anything like this in its Javadoc. My bias would then be to avoid adding this information there, since no other information is there.